ui/components/app/transaction-decoding/components/ui/accreditation/accreditation.component.js

```diff
@@ -23,14 +26,15 @@ const Accreditation = ({ fetchVia, address }) => {
   const AccreditationLink = () => {
     return (
       <>
-        <Typography
-          variant={TypographyVariant.H7}
+        <Text
+          variant={TextVariant.bodySm}
+          as="h6"
           className="accreditation__prefix"
-          boxProps={{ margin: 0 }}
+          margin={0}
```

Comment:
Shouldn't need this unless there is a margin being added by the `accreditation__prefix` className? If that is the case we should adjust it using the margin props
```suggestion
```